sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1375,6 +1375,9 @@ public void updateCompleteness(String caseUuid) {
 	}
 
 	@Override
+	@RolesAllowed({
```

Comment:
Why do we need the `CASE_EDIT` right for getting the data from the DB? 
`SAMPLE_VIEW` also should not be needed here

Reply:
CASE_EDIT is an error, locally I had CASE_VIEW annotation (which is also on the entire class but needs to be added again if the method has another annotation) and SAMPLE_VIEW because in case of logging in as an EXTERNAL LAB OFFICER the samples were not visible and in the log messsages appeared that the Client was unauthorized for getCaseDataByUuid method from CaseFacadeEjb and getPersonByUuid method from PersonFacadeEjb to view the samples.

Since yesterday https://github.com/hzi-braunschweig/SORMAS-Project/issues/8841 was fixed, the problem was solved ( I will delete the annotations related to this problem).